### PR TITLE
added tan to math.py in paddle/tensor

### LINKED ISSUE #15093: tan

### PR DESCRIPTION
**Pull request by Gadangadang**
Close #15093

### CODE REVIEW

**Review by YushaArif99 — COMMENTED** [review]
Hi @Gadangadang 

Is there a particular subtask this PR is referring to? The issue # you've provided is of a subtask you yourself created. As a contributor, you are only allowed to work on existing open tasks. If there is no such open task, I think its better to then close this PR and choose one of the functions in the open ToDo lists [here](https://github.com/unifyai/ivy/issues?q=is%3Aopen+is%3Aissue+label%3AToDo+label%3A%22PaddlePaddle+Frontend%22)

File: ivy/functional/frontends/paddle/tensor/math.py [hunk 1]
```diff
@@ -13,3 +14,10 @@ def sin(x, name=None):
 @to_ivy_arrays_and_back
 def cos(x, name=None):
     return ivy.cos(x)
+
+
+@to_ivy_arrays_and_back
+def tan(x, name=None):
+    x = ivy.astype(x, ivy.float64)
```
Comment (YushaArif99) [inline ivy/functional/frontends/paddle/tensor/math.py:21]: same goes for here as well. Hardcoding `ivy.float64` is not an elegant solution

**Comment by YushaArif99** [comment]
Hi @Gadangadang 

Are you still working on this PR? If not, we can then close this PR 🙂

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**